packages/saml/TEST_ADDITION.md

```
@@ -0,0 +1,64 @@
+# SAML Package Tests Added
+
+## Problem
+
+The CI/CD pipeline was failing because the `@authhero/saml` package had a test script but no test files:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)